docs/sources/set-up/install-agent-docker.md

```diff
@@ -21,10 +21,10 @@ Install Grafana Agent and get it up and running on Docker.
 docker run \
   -v /tmp/agent:/etc/agent/data \
   -v /path/to/config.yaml:/etc/agent/agent.yaml \
-  grafana/agent:v0.30.1
+  grafana/agent:latest
```

Comment:
I don't think we should change this unless we bump the version

Reply:
I'll set it back. We might have a new version when this is all said and done and we can make sure it is correct.